packages/fuels/tests/contracts.rs

```diff
@@ -1321,3 +1321,124 @@ async fn low_level_call() -> Result<()> {
 
     Ok(())
 }
+
+#[cfg(any(not(feature = "fuel-core-lib"), feature = "rocksdb"))]
+#[test]
+fn db_rocksdb() {
+    use fuels::accounts::fuel_crypto::SecretKey;
+    use fuels::accounts::wallet::WalletUnlocked;
+    use fuels::client::{PageDirection, PaginationRequest};
+    use fuels::prelude::DEFAULT_COIN_AMOUNT;
+    use fuels::prelude::{setup_test_provider, Config, DbType, ViewOnlyAccount};
+    use std::fs;
+    use std::str::FromStr;
+
+    let temp_dir = tempfile::tempdir()
+        .expect("Failed to make tempdir")
+        .into_path();
+    let temp_dir_name = temp_dir
+        .file_name()
+        .expect("Failed to get file name")
+        .to_string_lossy()
+        .to_string();
+    let temp_database_path = temp_dir.join("db");
+
+    tokio::runtime::Runtime::new()
+        .expect("Tokio runtime failed")
+        .block_on(async {
+            let wallet = WalletUnlocked::new_from_private_key(
+                SecretKey::from_str(
+                    "0x4433d156e8c53bf5b50af07aa95a29436f29a94e0ccc5d58df8e57bdc8583c32",
+                )
+                .unwrap(),
+                None,
+            );
+
+            const NUMBER_OF_ASSETS: u64 = 2;
+
+            let mut node_config = Config {
+                database_path: temp_database_path.clone(),
+                database_type: DbType::RocksDb,
+                ..Config::local_node()
+            };
+
+            node_config.manual_blocks_enabled = true;
+
+            let chain_config = ChainConfig {
+                chain_name: temp_dir_name.clone(),
+                initial_state: None,
+                transaction_parameters: Default::default(),
+                ..ChainConfig::local_testnet()
+            };
+
+            let (coins, _) = setup_multiple_assets_coins(
+                wallet.address(),
+                NUMBER_OF_ASSETS,
+                DEFAULT_NUM_COINS,
+                DEFAULT_COIN_AMOUNT,
+            );
+
+            let (provider, _) =
+                setup_test_provider(coins.clone(), vec![], Some(node_config), Some(chain_config))
+                    .await;
+
+            provider.produce_blocks(2, None).await?;
+
+            Ok::<(), Box<dyn std::error::Error>>(())
+        })
+        .unwrap();
+
+    // I had to terminate the runtime as I encountered an issue while attempting to start two nodes within the same runtime that utilize the same RocksDB.
```

Comment:
Do we need to keep this comment?

Reply:
https://github.com/FuelLabs/fuels-rs/pull/998#discussion_r1252143393
Maybe it is better to keep it.